jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -1011,8 +1050,7 @@ void java_object_factoryt::gen_nondet_init(
     assign.add_source_location() = location;
 
     assignments.add(assign);
-    // add assumes to obey numerical limits (given via
-    // java-assume-inputs-interval)
+    // add assumes to obey numerical restrictions
```

Comment:
:pick: Some documentation in the function doc would be useful:
```
Extra constraints are added to initialized objects, as specified in`object_factory_parameters`.
```

Reply:
What is the link supposed to point to?

Reply:
It's the field you get the options from.

Reply:
Added